README.md

```diff
@@ -1,4 +1,76 @@
-# StudentAPI-nodeJs
-* A restful API that perform CRUD operations on Students' and courses' data
-* student can take or drop courses
-* Link: https://danielstudent-api.herokuapp.com/api
+# Student Web Server
+
+Student Web Server is a RESTful API that manages student's and courses' information the API performs CRUD operations on student and course information, and returns the results in JSON format. The API can allow students to register for courses, and students can view their current courses.
+
+## Table of Contents
+
+- [Motivation](#motivation)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Testing](#api-testing)
+- [Tools](#tools)
+- [Heroku](#heroku)
+- [Authors](#authors)
+
+# Motivation
+
+Student Web Server was inspired by my college schedule builer, and I wanted to create a web application that would allow student to create my their own schedule by adding courses to their schedule or removing courses from their schedule.
```

Comment:
```suggestion
Student Web Server was inspired by my college schedule builder. I wanted to create a web application that would allow students to create their own schedules by adding or removing courses.
```